websites/L/Leboncoin/dist/metadata.json

```
@@ -0,0 +1,27 @@
+{
+	"$schema": "https://schemas.premid.app/metadata/1.7",
+	"author": {
+		"name": "frnk",
+		"id": "232904284004548608"
+	},
+	"service": "Leboncoin",
+	"description": {
+		"en": "Leboncoin is a french classified ads website where users and professionals can sell or buy products online and offline.",
+		"fr": "Leboncoin est un site français de petites annonces où des utilisateurs et des professionnels peuvent vendre ou acheter des produits en ligne et hors ligne."
+	},
+	"url": [
+		"www.leboncoin.fr",
+		"leboncoin.fr"
+	],
```

if you omit the `www.` you will be redirected, so
```suggestion
	"url": "www.leboncoin.fr",
```